sdk/core/azure-core/CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Release History
 
-## 1.1.0 (Unreleased)
+## 1.1.0 (2021-07-01)
 
 ### Bug Fixes
 
 - Fixed a memory leak issue in `Base64Encode()`. (A community contribution, courtesy of _[jorgen](https://github.com/jorgen)_)
 
+### Other Changes and Improvements
```

Comment:
```suggestion
### Other Changes
```


The headings we want to use are:
`Features Added`, `Breaking Changes`, `Bugs Fixed`, `Other Changes`
https://github.com/Azure/azure-sdk/blob/26e40bb0a9444eb7c7e57d0d1be4a6cf04f3fcbb/docs/policies/releases.md#change-logs

@antkmsft can you make the changes for the new CL entries for all three packages?